build.gradle

```diff
@@ -32,14 +32,19 @@ project.dependencies {
     else
     {
         // Use local version outside of TeamCity so as to not confuse IntelliJ
-        implementation(project(BuildUtils.getApiProjectPath(project.gradle))) { transitive = false }
+        BuildUtils.addLabKeyDependency(project: project, config: "implementation", depProjectPath: BuildUtils.getApiProjectPath(project.gradle), transitive: false)
     }
 
     // Issue 36184: Tests can't find selenium-ie-driver library when running on Windows on TeamCity
     // Workaround to ensure that the class-loader finds InternetExplorerDriver
     implementation "org.seleniumhq.selenium:selenium-ie-driver:${seleniumVersion}"
 }
 
+if (project.findProject(BuildUtils.getApiProjectPath(project.gradle)) != null)
+{
+    project.evaluationDependsOn(BuildUtils.getApiProjectPath(project.gradle))
+}
```

Comment:
I don't think this should be necessary.  Were you getting errors about, say, tasks or configurations from the api project not being defined that the test project relied on?  I don't think it does harm to have it here, but it's just one more thing for Gradle to do.

Reply:
Gone!